doc/conf.py

```diff
@@ -21,7 +23,28 @@
         "gmt": f">={required_gmt_version}",
     }
 )
+# Is a development version or not.
+isdev = "dev" in __version__ or __version__ == "unknown"
+# Some variables.
+repository = "GenericMappingTools/pygmt"
+repository_url = f"https://github.com/{repository}"
+doc_url = "https://pygmt.org/"
+if __commit__:
+    commit_link = f'<a href="{repository_url}/commit/{__commit__}">{__commit__[:8]}</a>'
+else:
+    commit_link = (
+        f'<a href="{repository_url}/releases/tag/{__version__}">{__version__}</a>'
+    )
+
+# Projection information.
+project = "PyGMT"
+author = "The PyGMT Developers"
+copyright = f"2017-{datetime.date.today().year}, {author}"  # noqa: A001
```

Comment:
We can use `f"2017-%Y, {author}"` after we requires sphinx>=8.1
(xref: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-copyright)